.gitignore

```diff
@@ -0,0 +1,53 @@
+# Jetbrains IDEs
+/.idea
+
+/__pycache__
```

Comment:
https://github.com/github/gitignore/blob/master/Python.gitignore is the (standard) Python .gitignore file